src/test/java/uk/gov/ida/eventemitter/Decrypter.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.ida.eventemitter;
+
+public interface Decrypter<T> {
```

Comment:
I think we only ever have a single implementation of this - do we need the Interface, or it is simpler to just use the concrete class?